src/repr/src/relation.rs

```diff
@@ -69,6 +69,13 @@ pub struct RelationType {
     /// A collection can contain multiple sets of keys, although it is common to
     /// have either zero or one sets of key indices.
     pub keys: Vec<Vec<usize>>,
+    /// Sets of (group, sum_column) indices, such that the following property holds:
+    /// `SELECT sum(sum_column) FROM relation GROUP BY <group>` is always 0 or 1.
+    /// This may sound like a niche property, but it amounts to saying "exploding by `sum_column` turns `group` into a key",
+    /// which is necessary for propagating Debezium key information.
+    // TODO [btv] - If in the future we start needing an unweildy amount of metadata to propagate Debezium key information,
+    // maybe we can do it in a hackier way (i.e., just have a `pre_debezium_keys` field here).
+    pub group_sum_keys: Vec<(Vec<usize>, usize)>,
```

Comment:
Do we still want to do this if we also have the `RelationExpr::Assert` thing?

Reply:
Nope